### ISSUE TITLE
Mangled text when using markdown table

### ISSUE THREAD

**Issue by gregtap** [n1k0/toctoc#5]
```# Foobar Backoffice

Interface to manage Foobar data.

https://backoffice.Foobar.com/

## Table of Contents

---

## Configuration

### Environment variables

Name | Description | Example Format
---- | ----------- | --------------
ALGOLIA_API_KEY | Algolia API key | Foobar
ALGOLIA_APP_ID | Algolia Application ID | Foobar
ALGOLIA_RECIPE_INDEX_NAME | Algolia index name | Foobar
ANALYTICS_DATABASE_HOST | analytics db: Mysql hostname | 127.0.0.1
ANALYTICS_DATABASE_NAME | analytics db: Mysql database name | Foobar
ANALYTICS_DATABASE_PASSWORD | analytics db: Mysql password | mypass
ANALYTICS_DATABASE_USER | analytics db: Mysql user name | myusername
DATABASE_HOST | default db: Postgres hostname | 127.0.0.1
DATABASE_NAME | default db: name | Foobar```

```
> toctoc README.md -w
```

```
# Foobar Backoffice

Interface to manage Foobar data.

https://backoffice.Foobar.com/

  ## Table of Contents

  - [Configuration](#configuration)
     - [Environment variables](#environment-variables)

---- | ----------- | --------------
ALGOLIA_API_KEY | Algolia API key | Foobar
ALGOLIA_APP_ID | Algolia Application ID | Foobar
ALGOLIA_RECIPE_INDEX_NAME | Algolia index name | Foobar
ANALYTICS_DATABASE_HOST | analytics db: Mysql hostname | 127.0.0.1
ANALYTICS_DATABASE_NAME | analytics db: Mysql database name | Foobar
ANALYTICS_DATABASE_PASSWORD | analytics db: Mysql password | mypass
ANALYTICS_DATABASE_USER | analytics db: Mysql user name | Foobar
DATABASE_HOST | default db: Postgres hostname | 127.0.0.1
DATABASE_NAME | default db: name | recipes_db
```

**Comment by n1k0** [comment]
Fixed by #6

**Comment by n1k0** [comment]
Released v0.3.1.
